dip-0011.md

```diff
@@ -68,11 +68,13 @@ that mining fees could be extremely variable. Registering an Identity would use
 more data than a standard transaction; if Dash blocks were full then the fees
 could be non-negligible and represent a barrier to entry into the system.
 
-Another big advantage is that the life cycle of an Identity exists directly
-inside the Platform consensus. If this was not the case then it would be very
+Another big advantage is that an Identity exists exclusively inside of 
+Platform's consensus. If this was not the case then it would be very
```

Comment:
The meaning here is that changes on an Identity exist inside the Platform consensus. I must say I prefer the first wording.

Reply:
Agree!

Reply:
https://github.com/dashpay/dips/pull/66/commits/8ff7a01879fb513154325208be5b6384e1c0e9f7